pagopa/io-app
# feat: [IOCOM-699] Add remote content banner

## Short description
This PR add `RemoteContentBanner` on message details in order to indicate that the message has remote content (It can change over time)

## List of changes proposed in this pull request
- used forwardRef in the component StatusContent
- added the `RemoteContentBanner` component
- updated MessageDetail in order to render the `RemoteContentBanner` component

## How to test
Using `io-dev-api-server` generate a message with remote content. Open the message detail and check that the content is displayed correctly.
